sdcard/os/archlinux.sh

```diff
@@ -155,3 +161,45 @@ allwinnerdownload(){
 umount_root(){
     umount $ROOT
 }
+
+amlogicformat(){
+
+  # See https://archlinuxarm.org/platforms/armv8/amlogic/odroid-c2
+	# 1. Zero the beginning of the SD card or eMMC module:
+	dd if=/dev/zero of=$SDCARD bs=1M count=8
+
+	# 2. and 3. Start fdisk to partition the SD card:
```

Comment:
Turns out that this code (2,3,4) is equal to `allwinnerformat`.
`allwinnerformat` should be renamed to `onepartitionformat` and `generalformat` to `twopartitionformat`